netbox_onboarding/tests/test_netbox_keeper.py

```diff
@@ -72,6 +75,30 @@ def setUp(self):
         self.ndk2.mgmt_ifname = "ge-0/0/0"
         self.ndk2.mgmt_pflen = 24
 
+        self.ndk3 = NetdevKeeper(self.onboarding_task2)
```

Comment:
Would be good to add comments to each of these new instances explaining their purpose, basically replicating the comments that you have below in the test case that uses these instances.